roles/rhel_subscribe/tasks/main.yml

```diff
@@ -41,7 +41,7 @@
     msg: 'Unable to register host with Red Hat Subscription Manager'
```

Comment:
This task seems unnecessary.  The registered tasks don't have failed_when: False, so they should be able to fail as-is.

We are also in a situation where this task would not work as desired since we're registering the same variable name twice.  We would need to use a unique registered name for rh_subscription for both types of registration and use an or statement or two separate fail tasks.

Reply:
Will remove this task and update the pr